charts/asset-manager/templates/nginx-configmap.yaml

```diff
@@ -210,9 +210,7 @@ data:
           # want to unset a request header sent TO S3 by nginx.
           proxy_set_header Authorization "";
 
-          # Add CloudFlare and Google DNS server to avoid "no resolver defined to resolve"
-          # errors when trying to connect to S3.
-          resolver 1.1.1.1 8.8.8.8 8.8.4.4;
+          resolver 172.20.0.10;  # kube-dns.kube-system.svc.cluster.local.
```

Comment:
according to the docs, this can be a hostname (not sure how it resolves it), would it make sense to set this to the hostname in case kube-dns isn't on that IP?

Reply:
Also, we aren't setting this in the router nginx config, and that instance also needs to resolve hostnames

Reply:
Oh cool, thanks for spotting that! Done.

Reply:
Sorry only just saw your second comment there! I'll check what's going on with the Router one. Good catch!